File: FSA.API/Models/TransactClaim.cs
using FSA.API.Models.Interface;
using System.ComponentModel.DataAnnotations;

namespace FSA.API.Models
{
    public class TransactClaim : ITransactClaim
    {
        [StringLength(50)]
        public string ReferenceNumber { get; set; }

        public decimal ClaimAmount { get; set; }
        [StringLength(25)]
        public string ReceiptNumber { get; set; }
        public decimal ReceiptAmount { get; set; }
        [StringLength(10)]
        public string ReceiptDate { get; set; }
    }
}

File: FSA Claim/ClientApp/src/app/services/data-classes/TransactClaim.ts
import { ITransactClaim } from "../interfaces/ITransactClaim";

export class TransactClaim implements ITransactClaim {
  referenceNumber: string = "";
  claimAmount: number = 0;
  receiptNumber: string = "";
  receiptAmount: number = 0;
  receiptDate: string = "";
}
